### PR TITLE
update quant_aware and quant_post for paddle version 2.0

### PR DESCRIPTION
**Pull request by slf12**
update quant_aware and quant_post for paddle version 2.0

### CODE REVIEW

File: paddleslim/quant/quanter.py [hunk 1]
```diff
@@ -257,6 +262,10 @@ def quant_post(executor,
             are under the path.
         quantize_model_path(str): The path to save quantized model using api
             ``fluid.io.save_inference_model``.
+        batch_generator(Python Generator): The batch generator provides 
+                calibrate data for DataLoader, and it returns a batch every
+                time. Note that, sample_generator and batch_generator, only one
```
Comment (wanghaoshuang) [inline paddleslim/quant/quanter.py]: Note that, sample_generator and batch_generator, only one should be set.

确定这语法对么？

File: paddleslim/quant/quanter.py [hunk 2]
```diff
@@ -279,6 +288,17 @@ def quant_post(executor,
                         "mul"].
         weight_bits(int, optional): quantization bit number for weights.
         activation_bits(int): quantization bit number for activation.
+	activation_quantize_type(str): quantization type for activation,
+                now support 'range_abs_max', 'moving_average_abs_max' and 'abs_max'.
+                This param only specifies the fake ops in saving quantized model.
```
Comment (wanghaoshuang) [inline paddleslim/quant/quanter.py]: 'param'不是正确的单词

File: paddleslim/quant/quanter.py [hunk 3]
```diff
@@ -257,6 +262,10 @@ def quant_post(executor,
             are under the path.
         quantize_model_path(str): The path to save quantized model using api
             ``fluid.io.save_inference_model``.
+        batch_generator(Python Generator): The batch generator provides
```
Comment (wanghaoshuang) [inline paddleslim/quant/quanter.py:265]: Python Generator
类型这么写是专业的么？

**Review by wanghaoshuang — COMMENTED** [review]
文档的语法整体写的有点不严谨，希望再斟酌下。

File: paddleslim/quant/quanter.py (hunk 1 above)
Comment (slf12) [reply paddleslim/quant/quanter.py]: update

File: paddleslim/quant/quanter.py (hunk 2 above)
Comment (slf12) [reply paddleslim/quant/quanter.py]: update

File: paddleslim/quant/quanter.py (hunk 3 above)
Comment (slf12) [reply paddleslim/quant/quanter.py:265]: https://docs.python.org/2/library/stdtypes.html#generator-types,  Generator Function or Generator?

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**